runelite-client/src/main/java/net/runelite/client/ui/ClientUI.java

```diff
@@ -680,6 +726,20 @@ private void updateFrameConfig(boolean updateResizable)
 			return;
 		}
 
+		if (config.usernameInTitle() && (client instanceof Client))
+		{
+			final Player player = ((Client)client).getLocalPlayer();
+
+			if (player != null && player.getName() != null)
+			{
+				frame.setTitle(properties.getTitle() + " - " + player.getName());
+			}
+		}
+		else
+		{
+			frame.setTitle(properties.getTitle());
```

Comment:
I'd like to see the string defined before doing the `config.usernameInTitle()` check, and this else block removed. That way, only one `frame.setTitle()` call is needed.

Reply:
but in first if the .setTitle is not always called, because title do not needs updating (as there are no valid data to update the title with)

Reply:
Hm... I suppose that's true. Would be nice if we could find a way to avoid the double-call, but I guess that's not possible here.